ports/stm32f4/common-hal/neopixel_write/__init__.c

```diff
@@ -40,6 +42,9 @@ uint32_t next_start_tick_us = 1000;
 #define MAGIC_800_T0H  2800000  // ~0.36 us -> 0.44 field
 #define MAGIC_800_T1H  1350000  // ~0.74 us -> 0.84 field
```

Comment:
Did you test this after you removed the nops? I'd expect it to change the timing slightly (more so on the slower clock speeds.)

Reply:
Yes, they're still within tolerance - works on the PYB Nano, Discovery. I'll give dynamic code another shot once the F401 gets added, since I'll have 3 test cases at that point.